cookbook/case_studies/feature_engineering/feast_integration/feast_workflow.py

```diff
@@ -60,6 +62,22 @@
 DATA_CLASS = "surgical lesion"
 
 
+@task
+def create_bucket(bucket_name: str):
+    client = Minio(
+        "localhost:30084",
+        access_key=os.environ["FLYTE_AWS_ACCESS_KEY_ID"],
+        secret_key=os.environ["FLYTE_AWS_SECRET_ACCESS_KEY"],
+        secure=False,
+    )
```

Comment:
Can you check how we access the env vars in cookbook/case_studies/feature_engineering/feast_integration/feast_dataobjects.py?